tools/wptserve/wptserve/response.py

```diff
@@ -711,26 +701,31 @@ def write_header(self, name, value):
         if not self._status_written:
```

Comment:
Add `:return: A boolean indicating whether the write succeeds` to the end of the docstring.